commons/zenoh-keyexpr/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2022 ZettaScale Technology
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+# which is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+#
+# Contributors:
+#   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+#
+[package]
+rust-version = { workspace = true }
+name = "zenoh-keyexpr"
+version = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+license = { workspace = true }
+categories = { workspace = true }
+description = "Internal crate for zenoh."
+
+[features]
+default = ["std"]
+std = ["zenoh-result/std", "rand?/std", "rand?/std_rng"]
+test = ["rand"]
+
+[dependencies]
+keyed-set = { workspace = true }
+rand = { workspace = true, features = ["alloc", "getrandom"], optional = true }
```

Comment:
Should `rand` be part of `[dependencies]` as well or only of `[dev-dependencies]`?

Reply:
It's a dependency if we want the KeFuzzer to be available to other crates

Reply:
Right, it optionally included it the `test` features is enabled for `zenoh-keyexpr`. Fine with that.